Makefile

```diff
@@ -110,8 +110,6 @@ cfmt:
 	find . '(' -name '*.h' -o -name '*.c' ')' ! -path './vendor/*'  -exec clang-format -i {} \+
```

Comment:
I think the `fmt` and `cfmt` taret can be removed as well since we don't have any C code left on the repo.